### PR TITLE
Type one less sudo command by using 'cat' and a redirect

### PR DESCRIPTION
**Pull request by DylanGraham**

### CODE REVIEW

**Comment by spaletta** [comment]
You don’t need that `tee`, a simple shell redirect (`> $HOME/.kube/config`) is enough. Your own shell sets up the redirect, so the file opening / creation happens with the correct user’s identity and permissions.
The change is also problematic, because – in contrast to current instructions – the whole KubeConfig will also be dumped in the user’s face.

**Comment by spiffxp** [comment]
/ok-to-test

(FWIW, I agree with @spaletta)

**Comment by luxas** [comment]
@DylanGraham Welcome to the Kubernetes community! Thanks for your contribution.
Sorry it took us long to notice this PR. We'll take a look and discuss

cc @jamiehannaford @jbeda @lukemarsden @pipejakob @timothysc PTAL

**Comment by jamiehannaford** [comment]
LGTM

**Comment by timothysc** [comment]
I don't see why you don't need the chown...

File: cmd/kubeadm/app/cmd/init.go [hunk 1]
```diff
@@ -51,8 +51,7 @@ var (
 		To start using your cluster, you need to run (as a regular user):
 
 		  mkdir -p $HOME/.kube
-		  sudo cp -i {{.KubeConfigPath}} $HOME/.kube/config
-		  sudo chown $(id -u):$(id -g) $HOME/.kube/config
+		  sudo cat {{.KubeConfigPath}} > $HOME/.kube/config
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/init.go]: I'm worried that this breaks the wall of safety we had with `cp -i`... this will just overwrite whatever's there

**Review by luxas — COMMENTED** [review]
Not sure about this tbh

**Pull request closed without merging**